.github/workflows/release.yml

```
@@ -62,12 +66,24 @@ jobs:
         if: ${{ github.event_name == 'pull_request' }}
       - run: 'echo -e "env_variables:\n  ENV_NAME: \"prd\"" >> app.yaml'
         if: ${{ github.event_name == 'push' }}
+      - uses: google-github-actions/setup-gcloud@v0.2.1
+        with:
+          project_id: 'hato-atama'
+      - id: 'auth'
+        name: 'Authenticate to GCP'
```

```suggestion
        name: 'Authenticate to Google Cloud'
```

GCPからGoogle Cloudに改名したはず。

---

勘違いっぽい。